deployment/GP03FLMB_Deploy.csv

```diff
@@ -143,3 +143,21 @@ SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-
 SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
 SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0180,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
 SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
+SKQ202208S,,,,GP03FLMB-RIM01-00-SIOENG000,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGCON-SIOCON-14876,50.3,-144.51,,1507,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-ADCPSL007,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-ADCPSL-24661,50.3,-144.51,,500,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG060,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-13643,50.3,-144.51,,30,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG061,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-11609,50.3,-144.51,,40,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG062,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-12309,50.3,-144.51,,60,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG063,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-12406,50.3,-144.51,,90,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG064,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-12587,50.3,-144.51,,130,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG065,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-11593,50.3,-144.51,,180,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG066,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-11637,50.3,-144.51,,250,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG067,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-11535,50.3,-144.51,,350,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOG068,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOG-12310,50.3,-144.51,,500,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOH069,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOH-12389,50.3,-144.51,,750,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOH070,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOH-11683,50.3,-144.51,,1000,4176,,,
+SKQ202208S,,,,GP03FLMB-RIM01-02-CTDMOH071,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-CTDMOH-10264,50.3,-144.51,,1500,4176,,,
+SKQ202208S,,,,GP03FLMB-RIS01-00-SIOENG000,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGCON-SIOCON-13284,50.3,-144.51,,30,4176,,,
+SKQ202208S,,,,GP03FLMB-RIS01-03-DOSTAD000,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-DOSTAD-00456,50.3,-144.51,,30,4176,,,
+SKQ202208S,,,,GP03FLMB-RIS01-04-PHSENF000,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-PHSENF-P0183,50.3,-144.51,,30,4176,,,
+SKQ202208S,,,,GP03FLMB-RIS01-05-FLORTD000,9,1,2022-05-11T00:00:00,,CGMGP-03FLMB-00009,,CGINS-FLORTD-01102,50.3,-144.51,,30,4176,,,
```

Comment:
serial numbers good, but noting that water depth on website is 4145 meters

Reply:
these are just placeholder 'dummy' depths.  will be updated after deployment.